### PR TITLE
Maksze telemetry bundle

### PR DESCRIPTION
**Pull request by maksze**
| Q             | A
| ------------- | ---
| License       | MIT

Maksze Telemetry Bundle

### CODE REVIEW

**Comment by maksze** [comment]
@Nyholm what's wrong with it PR?

**Comment by Nyholm** [comment]
It looks like you have not made any changes. You removed all files here: https://github.com/symfony/recipes-contrib/pull/735/commits/85bb79f2954242a16fcc743220961bdac55e4b86


Did you forget to add 0.3?

**Comment by maksze** [comment]
my bad( i forget to add 0.3.

File: maksze/telemetry-bundle/0.3/config/packages/maksze_telemetry.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+maksze_telemetry:
+    yandex_metrika:
+        #- {id: any_id}
+    yandex_webmaster:
+        #- {id: any_id}
+    google_search_console:
+        #- {id: any_id}
+    google_analytics:
+        #- {id: any_id}
```
Comment (Nyholm) [inline maksze/telemetry-bundle/0.3/config/packages/maksze_telemetry.yaml]: You should consider using environment variables here.

Comment (maksze) [reply maksze/telemetry-bundle/0.3/config/packages/maksze_telemetry.yaml]: Yes, of course

File: maksze/telemetry-bundle/0.4/config/packages/maksze_telemetry.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# For more information about the bundle settings visit:
+# https://github.com/maksze/MaksZeTelemetryBundle/blob/master/README.md
+
+maksze_telemetry:
+    yandex_metrika:
+        #- {id: '%env(YANDEX_METRIKA_ID)%'}
+    yandex_webmaster:
+        #- {id: '%env(YANDEX_WEBMASTER_ID)%'}
+    google_search_console:
+        #- {id: '%env(GOOGLE_SEARCH_CONSOLE_ID)%'}
```
Comment (Nyholm) [inline maksze/telemetry-bundle/0.4/config/packages/maksze_telemetry.yaml]: No need to use comments here, right?

I would assume a user have at least two of these values ready when installing the bundle.

Comment (maksze) [reply maksze/telemetry-bundle/0.4/config/packages/maksze_telemetry.yaml]: I do not agree, in Russia we often use Yandex services than Google

**Comment by maksze** [comment]
@Nyholm Thank you

**Pull request merged**